ci/install_openebs.sh

```diff
@@ -1,17 +1,10 @@
 #!/usr/bin/env bash
 
-#OPENEBS="$(https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-operator.yaml)"
-#CASTEMPLATE="${https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-cas-templates-pre-alpha.yaml}"
-#OPENEBSVOLUME="${https://raw.githubusercontent.com/openebs/openebs/master/k8s/demo/pvc-standard-jiva-default.yaml}"
-
 echo "--------------------Installing openebs operator---------------------------"
-sleep 10
+sleep 5
+kubectl create clusterrolebinding cluster-admin:kube-system --clusterrole=cluster-admin --serviceaccount=kube-system:default
```

Comment:
What is this clusterrolebinding used for?

Reply:
No use .... need to delete